src/App/App.js

```diff
@@ -19,10 +18,11 @@ class App extends Component {
     return (
       <NavigationContainer>
         <Stack.Navigator screenOptions={{
-          headerShown: false
+          headerShown: true
```

Comment:
Do we need to show header? I removed it because it seemed un-needed - let me know if there is something I'm missing.